devel/loadtest_collect/loadtest_collect_corpus.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script collects slurps and URLs (post-optimization, if possible), of
+# some websites with the help of phantomjs.
+
+function usage {
+  echo "Usage: loadtest_collect/loadtest_collect_corpus.sh pages.txt out.tar.bz2"
+}
+
+set -u  # exit the script if any variable is uninitialized
+set -e
+
+if [ $# -ne 2 ]; then
+  usage
+  exit 1
+fi
+
+if [ -d devel ]; then
+  cd devel
+fi
+if [ ! -d loadtest_collect ]; then
+  echo Run this script from the top or devel/ directories
+  exit 1
+fi
+
+if [ ! $(which phantomjs) ]; then
+  sudo apt-get install phantomjs
```

Comment:
about echoing what you are doing first.  If you run this from CentOS I think it'll just fail saying "apt-get command not found", and the user won't know it's just trying to install phantom without debugging the script.  With the echo at least the CentOS user could just install it manually.